src/app.jsx

```diff
@@ -12,6 +13,55 @@ import routes from './routes';
 import analytics from './services/analytics';
 
 const App = () => {
+  // Code for setting up color schemes
+
+  const { mode, setMode } = useColorScheme();
+  // mode listener code from
+  //https://dev.to/gustavogomez/manage-dark-mode-in-reactjs-with-listener-to-detect-dark-mode-in-the-browser-3120
+
+  let colorMode = localStorage.getItem('colorMode') ?? 'system';
```

Comment:
The 'colorMode' local storage key should be a constant, probably defined in a service, that can be imported where needed.

Reply:
I ended up storing the key in theme.ts, since that's where the other palette information and settings are stored, rather than making a new service to only store the one constant.  If there is somewhere that makes more sense to store this, let me know.